internal/broker/bind_get_test.go

```diff
@@ -45,4 +52,39 @@ func TestGetBinding(t *testing.T) {
 		errorResponse := apiErr.ErrorResponse().(apiresponses.ErrorResponse)
 		require.Equal(t, "Binding expired", errorResponse.Description)
 	})
+
+	t.Run("should return 404 code for the binding of deprovisioned/suspended instance", func(t *testing.T) {
```

Comment:
Please consider changing test case name to "should return 404 code for the EXPIRED binding of deprovisioned/suspended instance

Reply:
Expiration is not important.

Reply:
Corrected `expiredBinding` to `binding`